scheduler/scheduler/config/configs.go

```diff
@@ -0,0 +1,72 @@
+package config
+
+var SchedulerConfigs = map[string]string{
+	"default":      defaultConfig,
+	"local.local":  localLocal,
+	"local.memory": localMemory,
+}
+
+const defaultConfig = `
+{
+	"Cluster": {
+	  "Type": "memory"
+	},
+	"Workers": {
+	  "Type": "local"
+	},
+	"SchedulerConfig": {
+	  "Type": "stateful",
+	  "MaxRetriesPerTask" : 0,
+	  "DefaultTaskTimeout" : "30m"
+	},
+	"SagaLog": {
+	  "Type": "memory"
+	}
+}
+`
+
+// localLocal config for local.local - !!! make sure this constant is added to SchedulerConfigs map above !!!
```

Comment:
is the !!! part needed in this comment?

Reply:
to draw attention to it - if the config is not added to the map, it will not be available.

Reply:
can we please add a comment to the map above that indicates that is what defines what configs are available

Reply:
sure - but it seems self-evident that it would need to be there imo

Reply:
yeah - mainly as opposed to commenting the each of the config definitions separately

Reply:
oh weird idk how these 2 convos got merged. wasn't responding to you drew! but yeah would rather have the one comment than the multiple

Reply:
I also wanted to comment on each config definition separately incase someone cuts and paste's the definition....hopefully when they update the pasted comment they will see the message to also add the new config up top of the file.